pyproject.toml

```diff
@@ -40,21 +40,19 @@ include = [
 
 [tool.poetry.dependencies]
 python = "^3.9"
-rustsim = "0.1.10"
-kgx = "^2.0.8"
+bmt = "^1.0.15"
 jsonschema = "^4.17.0"
 PyYAML = "^6.0"
 requests = "^2.28.1"
 pydantic = "^1.8.0"
-numpy = "^1.23.5"
 urllib3 = "^1.26.15"
+pytest-cov = "^4.0.0"
```

Comment:
Why are `pytest` and `pytest-cov` in the package dependencies? Could we put them as optional dependencies?

Reply:
after some debugging these should be optional dependencies now.  they will install with a direct install of the package itself, but should be removed from downstream installations of this package via pip, etc.  However, during debugging this chain of dependencies we noticed that [prefixcommons needs a tweak](https://github.com/prefixcommons/prefixcommons-py/pull/29).  That will take a few days to percolate through (and if you are pinning this dependency, reasoner-validator, you might not see it until we release again).